WalletWasabi/Legal/Assets/LegalDocumentsGingerWallet.txt

```diff
@@ -0,0 +1,484 @@
+Last Updated: May 28, 2024
+
+
+Version Number: 5
+
+
+=========================================
+
+DISCLAIMER
+
+=========================================
+
+
+================================================================
+
+PROHIBITION OF ACCESS FROM THE USA, ITS STATES, AND TERRITORIES
+
+================================================================
+
+
+  Users located in the United States of America, including its states and territories, are hereby prohibited from accessing this platform. InvisibleBit LLC does not provide services to individuals located in these regions and shall not be liable for any consequences resulting from unauthorized access. By accessing this platform, users outside of the United States confirm that they are not a national of or located within the United States or its territories and agree to abide by this restriction.
+
+
+=========================================
+
+SUMMARY
+
+=========================================
+
+
+    The GingerPrivacy team is committed to being fully transparent to its users, from its source code to the legal aspects of its services. The purpose of Ginger Wallet is to provide privacy for its users’ Bitcoin transactions. This means that with the use of the wallet, we would like to enable the users to make Bitcoin payments in such a way that their spending history, wallet balances, etc. are not visible to the receiving party. This does not necessarily mean that the user stays anonymous since the receiving party might have to identify the sending party.
+
+    Our goal is to minimize the unintentional leaking or disclosure of private financial information to third parties and the receiver of any transaction resulting from the use of the Ginger Wallet. The protection of your privacy and your personal data is our highest priority, which also defines the essence of our services.
+
+    One instance of the Ginger Wallet CoinJoin coordinating server is operated by a legal entity, InvisibleBit LLC. Thus, to legally provide our services to our clients, we have defined the framework and the rules that both parties must follow to protect each other.
```

Comment:
I am not sure about this change, I think there is no such thing as Ginger Wallet Coinjoin coordinator.

```suggestion
    One instance of the WabiSabi CoinJoin coordinating server is operated by a legal entity, InvisibleBit LLC. Thus, to legally provide our services to our clients, we have defined the framework and the rules that both parties must follow to protect each other.
```

Reply:
Yes, there is. It's just it is compatible with the Wasabi.

Reply:
I am talking about this sentence:  One instance of **the Ginger Wallet CoinJoin coordinating server** is operated by a legal entity,

What we are running is a WabiSabi coinjoin coordinator (that can be run by anyone else). It is not a Ginger Wallet coinjoin coordinator. Ginger Wallet is the client that connects to a WabiSabi coinjoin coordinator.
Anyone can change the coordinatoruri to any other coordinator that coordinates WabiSabi cjs.

Wasabi (and Gigner) is compatible with WabiSabi coordinators (not Wasabi Wallet coordinator).
Samurai is compatible with Whirpool coordinator (not Samurai wallet coordinator).
and so.